common/models/ClientCard.ts

```diff
@@ -44,6 +44,7 @@ export namespace CoachClientActions {
     export const Helper = new EnumStringHelper<CoachClientActions>(CoachClientActions);
 }
 
+// TODO: allow toggle to a shorter period for testing/development
```

Comment:
Is this TODO meant to be completed in this PR?

Reply:
@cara-wong. Added a feature to account for this - see the latest changes